src/components/Dao/DaoHistoryPage.tsx

```diff
@@ -34,24 +82,41 @@ class DaoHistoryPage extends React.Component<IProps, null> {
     });
   }
 
+  onSearchChange = async (e: any) => {
```

Comment:
Is there a reason why this function is `async`?

Reply:
This is an artifact of a previous change, but since that has been deleted, we can remove the async.